_data/design-status.yml

```diff
@@ -0,0 +1,12 @@
+stable:
+ name: stable
+ class: stable
+ description: 'The Helix team has a high degree of confidence in this design and considers it low risk and ready for production implementation.'
+in-progress:
+ name: in progress
+ class: 'in-progress'
+ description: 'The Helix team has moderate confidence in this design and considers it medium risk for implementation.  The pattern will likely undergo some changes in the near future.'
+deprecated:
+ name: deprecated
+ class: 'deprecated'
+ description: 'This design is going to be phased out. Helix considers this high risk and the item should no longer be implemented.'
```

Comment:
s/`Helix considers this high risk and the item should no longer be implemented.`/`The Helix team considers this design high risk, and this design should no longer be implemented.` or `The Helix team considers this design high risk; do not implement this design.`